zeus/monitor/power.py

```diff
@@ -117,7 +117,7 @@ def __init__(
         gpu_indices: list[int] | None = None,
         update_period: float | None = None,
     ) -> None:
-        """Initialize the power monitor.
+        """Initialize the power monitor. Initialization should not be done in global scope due to python's protection.
```

Comment:
```suggestion
        """Initialize the power monitor.
        
        Initialization should not be done in global scope due to python's protection.
```
Put a ref to doc. Same for energy.py.

Reply:
Not sure what you meant. Which doc are you talking about?

Reply:
Python official documentation saying that global variable can cause problems.